main.py

```diff
@@ -50,45 +50,30 @@ def main():
         # Set the input_file variable to the 2nd argument (input file path).
         input_file = sys.argv[1]
 
+        if not (os.path.isfile(input_file)):
+            print("ERROR: Path not valid.")
+
         # If the path is a legitimate file...
-        if (os.path.isfile(input_file)):
-            # Loop until break is called.
-            while True:
-                # Obtain user input.
-                to_trim = input("Do you want to trim this file? (y/n)\n")
-                if not (to_trim == "y" or to_trim == "n"):
-                    print("Invalid input, please enter either 'y' or 'n'.")
-                else:
-                    break
+        else:
+            to_trim = set_trim_status()
+            to_watermark = set_watermark_status()
+            # Ask user for their preferred output filename/filepath.
+            output_path = set_output_path(input_file)
 
-            # Get aspect ratio, resolution, and watermark image.
-            aspect_ratio, resolution = aspect_ratio_and_resolution(input_file)
+            # Get aspect ratio and watermark image.
+            aspect_ratio = stream_aspect_ratio(input_file)
```

Comment:
The `resolution` is never actually used, so I refactored the function not to return it.

It's good practice for a function to have a single purpose, where practical - a function with an `and` in the name is a sign that it needs to be refactored, IMO. (I might have broken this out into two functions if we actually needed `resolution`).    
(apologies if I'm stating the obvious here!)

Reply:
No, please do. Most of this is James work which I'm struggling to understand. MANY THANKS!! :)